### PR TITLE
put version attribute to use - 2.1.0

### PR DESCRIPTION
**Pull request by kplimack**
## 2.1.0
* use build-essential to install gcc,make, etc
* rubocop (.rubocop.yml added and files corrected)
* use generic centos-6 VM to test on, not internal sendgrid images
* alphabetize attributes
* replace ruby_block with file resource with sensitive
* add `Debian 8` support
* add `Centos 7` support
* add `travis-ci` config, Gemfile, etc
* allow for orchestrator package source/checksum to be defined and overridden

Signed-off-by: Jake Plimack <jake.plimack@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,57 +1,66 @@
-# 2.0.2
+# orchestrator
+
+## 2.1.0
+* use build-essential to install gcc,make, etc
+* rubocop (.rubocop.yml added and files corrected)
+* use generic centos-6 VM to test on, not internal sendgrid images
+* alphabetize attributes
+* replace ruby_block with file resource with sensitive
```
Comment (silviabotros) [inline CHANGELOG.md:8]: add a line for the orchestrator version attribute?

**Comment by silviabotros** [comment]
besides my one comment on the CHANGELOG, I am cool with this, Thanks Jake.

**Comment by kplimack** [comment]
link to passing build: https://travis-ci.org/kplimack/chef-orchestrator/builds/284412841

**Comment by kplimack** [comment]
sorry about the 1248324 commits, it took me a lot of tries to learn how to make travis happy

File: .kitchen.yml [hunk 2]
```diff
@@ -8,16 +8,25 @@ driver:
...
 platforms:
   - name: centos-6.6
-    driver_config:
-      box: sendgrid_centos-6_chef-11
-      box_url: http://repo.sjc1.sendgrid.net/images/vagrant/metadata/sendgrid_centos-6_chef-11.json
+    attributes:
+      orchestrator:
+        package:
+          version: 2.1.5-1
+          url: 'https://github.com/kplimack/orchestrator-rpm/raw/master/orchestrator-2.1.5-1.x86_64.rpm'
```
Comment (logikal) [inline .kitchen.yml:24]: did we get shlomi to re-upload this, @silviabotros ?

Comment (logikal) [reply .kitchen.yml:24]: Also at https://github.com/silviabotros/chef-orchestrator/pull/9/files#diff-c36c03b7f2ee2bb67c50a7c931a63a0eR22

File: README.md [hunk 3]
```diff
@@ -25,15 +25,31 @@ Tested with
...
+    <td>nil | string</td>
+    <td>Checksum for the file provided by URL</td>
+    <td><tt>nil</tt></td>
   </tr>
 </table>
 
 ## Usage
 
+### Installation
+Starting in version _2.1.0_, the orchestator cookbook now respects the `version` attribute. If set to `nil` (the default), the cookbook will perform the same steps as previous versions--always upgrade to the latest version available via the repos available on a node.
```
Comment (logikal) [inline README.md]: `orchestrator` is spelled wrong here.

File: README.md [hunk 4]
```diff
@@ -25,15 +25,31 @@ Tested with
...
+Setting this attribute to a specific package version will result in that version being installed. If you want to install the package from an internal source, you can also specify the url and optional checksum to download the orchetrator package.
```
Comment (logikal) [inline README.md]: `orchestrator` is also spelled wrong here.

File: .kitchen.yml (hunk 2 above)
Comment (silviabotros) [reply .kitchen.yml:24]: nope...Shlomi is on PTO till next week....we are on our own with the package till he resolves the build issues in 3.x

**Review by logikal — CHANGES_REQUESTED** [review]

**Pull request merged**